strawberryfields/backends/base.py

```diff
@@ -506,6 +506,25 @@ def measure_fock(self, modes, shots=1, select=None, **kwargs):
         """
         raise NotImplementedError
 
+    def measure_threshold(self, modes, shots=1, select=None, **kwargs):
+        """Measure the given modes in the thresholded Fock basis, i.e., zero or nonzero photons).
+
+        .. note::
+
+            When :code:``shots == 1``, updates the current system state to the conditional state of that
```

Comment:
Should we remove these lines (also for `measure_fock`)?

Reply:
These should remain, as this is the abstract base class of the backend API, and serves as instructions for new developers on how this method should act, _if_ they have the ability to calculate the conditional state of the system after measurement.